src/utils/context.jsx

```diff
@@ -26,11 +30,13 @@ const context = function context(Component, params) {
       }
       let slide = 0;
       if (this.props.params && "slide" in this.props.params) {
-        slide = this.props.params.slide;
+        slide = parseInt(this.props.params.slide);
```

Comment:
We don't explicitly need it; But can we either use `Number` or pass in our radix here, `parseInt(this.props.params.slide, 10)`